.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,56 @@
+---
+name: File an issue
+about: Create a new issue for Positron.
+---
+
+<!--
+
+Thanks for taking the time to file an issue!
+
+Take a look at our guidance on feedback and issues:
+https://github.com/posit-dev/positron/wiki/Feedback-and-Issues
+
+Most feedback should start in GitHub Discussions rather than as an issue:
+https://github.com/posit-dev/positron/discussions
+
+To avoid creating duplicates, please take the time to search our existing issues:
+https://github.com/posit-dev/positron/issues
+If you find an existing bug or feature request, please gives it a thumbs-up reaction or
+additional information to help us prioritize the issue.
+
+-->
+
+## System details:
+
+Positron and OS details:
+
+<!-- Copy from the Positron > About menu -->
+
+Language details:
+
+<!-- What language are you using? e.g. R 4.4.1 or Python 3.11.6 -->
+
+## Describe the issue:
+
+
+## Steps to reproduce the issue:
+
+<!-- For example, are you using a Jupyter notebook, R script, Quarto document, etc? -->
+
+1.
+2.
+
+
+## Expected or desired behavior:
+
+
+## Were there any error messages in the UI, Output panel, or Developer Tools console?
+
+<!--
+
+Open the Developer Tools console by running the Developer: Toggle Developer Tools command from the Command Palette.
+A guide to locating the correct Output channel can be found here: https://github.com/posit-dev/positron/wiki/Troubleshooting
```

Comment:
Can we make this a 2 bullet list? Right now to me it doesn't read like 2 things we want you to do. maybe:

```
- Open the Developer Tools console by running the `Developer: Toggle Developer Tools` command from the Command Palette.
- Open the relevant Output Channel logs by utilizing this guide https://github.com/posit-dev/positron/wiki/Troubleshooting
```